### PR TITLE
Docs: Browser Aliases topic added

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+The following is the list of browser aliases.
+
+* `chrome` - Google Chrome
+* `chromium` - Chromium
+* `edge` - Microsoft Edge
+* `ff` - Mozilla Firefox
+* `ie` - Internet Explorer
+* `opera` - Opera
+* `safari` - Safari
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md]: I prefer table with 'Browser | Alias' columns.

**Comment by inikulin** [comment]
\r-

File: docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md (hunk 1 above)
Comment (VasilyStrelyaev) [inline docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md]: Right!

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
It seems useful to \cc @DevExpress/testcafe for documentation but you wouldn't know when to merge... 
And, also... Does anyone besides @inikulin, honestly, reads all this?

**Comment by VasilyStrelyaev** [comment]
If few - maybe it will be more useful to have the team review the docs after everything is finished?

**Comment by inikulin** [comment]
BTW, we need a cross-link here to `--list-browsers` option in CLI that lists aliases of the available browsers

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
ping

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
Anyone else?
(I'm bored, I want to go ahead)

**Comment by inikulin** [comment]
@VasilyStrelyaev I have an idea how we can deal with this burden. Ping me tomorrow, please.

**Comment by kirovboris** [comment]
lgtm

**Comment by miherlosev** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
@inikulin cool
I think someone can already merge

**Comment by AlexanderMoskovkin** [comment]
done)

**Comment by VasilyStrelyaev** [comment]
thanks!

**Pull request merged**